tests/Sdk/Unit/Metrics/ValueRecorderTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Unit\Metrics;
+
+use InvalidArgumentException;
+use OpenTelemetry\Sdk\Metrics\ValueRecorder;
+use PHPUnit\Framework\TestCase;
+
+class ValueRecorderTest extends TestCase
```

Comment:
Looks like all tests except the last one follow the same pattern. I would suggest to move data out to a `@dataProvider`. This would let you have only one test with a bunch of test data instead of 8 similar tests